src/server/status_server.rs

```diff
@@ -240,6 +339,9 @@ impl StatusServer {
                             (Method::GET, "/status") => Box::new(ok(Response::default())),
                             (Method::GET, "/pprof/profile") => Self::dump_prof_to_resp(req),
                             (Method::GET, "/config") => Self::config_handler(config.clone()),
+                            (Method::GET, "/rsperf/start") => Self::start_rs_profiler(req),
+                            (Method::GET, "/rsperf/stop") => Self::stop_rs_profiler(),
+                            (Method::GET, "/rsperf/report") => Self::report_from_rs_profiler(),
```

Comment:
How about adding perf duration? 

I think golang pprof is good example https://golang.org/pkg/net/http/pprof/

Reply:
+1
should not use start and stop manually

Reply:
+2

Reply:
+3

Reply:
+4

Reply:
Now, the only HTTP interface left was `/rsperf/profile?frequency=50&seconds=10`!